types/gatherer.d.ts

```diff
@@ -20,6 +20,7 @@ declare global {
       options?: object;
       /** Push to this array to add top-level warnings to the LHR. */
       LighthouseRunWarnings: Array<string>;
+      baseArtifacts: BaseArtifacts;
```

Comment:
we might want to play around with `Readonly` for this, but that can be something for another day :)